Change analystic ID, update cytosu and add soyokaze! (#18)
Rurusetto/rurusetto: content/posts/soyokaze/index.md

---
title: "soyokaze!"
img: "soyokaze-logo.png"
date: 2021-04-20

> summary: "An osu! ruleset mimicking Genshin Impact&rsquo;s Ballads of Breeze mini-game."

Reviewer: ```suggestion
summary: "An osu! ruleset mimicking Genshin Impact's Ballads of Breeze mini-game."
```
I am not sure what happened here but I think this is wrong

Author: It's HTML numberic code for curly quote.

---

# status

- Status : Active development
- Last Update : 2021.419.1
- Can import and play without error : Yes

# overview

- Creator : [Alden Wu](https://github.com/goodtrailer)
- Availability : Available publicly
- GitHub Repositories : [goodtrailer/soyokaze](https://github.com/goodtrailer/soyokaze)

Soyokaze! is a custom ruleset mimicking Genshin Impact's Ballads of Breeze mini-game. The ruleset is developed by Alden Wu.

# gameplay

> Primarily in soyokaze! have 2 rhythm rings. A rhythm ring is like DPAD, it have 4 side (north, ease, south and west). A note will gradually be appear at the four side of the ring and then disappear. Player must press a note in time before it disappear.

Reviewer: ```suggestion
Primarily in soyokaze! the player has 2 rhythm rings. A rhythm ring is like DPAD, it have 4 side (north, east, south and west). A note will appear at one of the four sides of both rings and then disappear. The player must press the note in time before it disappears using the keyboard button assigned for this specific note placement.
```

# controls

Can only use keyboard to play.

### default controls

- Button 1 (DPAD-L ↑) - `W`
- Button 2 (DPAD-L ←) - `A`
- Button 3 (DPAD-L ↓) - `S`
- Button 4 (DPAD-L →) - `D`
- Button 5 (DPAD-R ↑) - `I`
- Button 6 (DPAD-R ←) - `J`
- Button 7 (DPAD-R ↓) - `K`
- Button 8 (DPAD-R →) - `L`

# mods

### difficulty reduction

- ![Easy Icon](mod-icon/easy-mod.png) Easy

>   - Larger circles, more forgiving HP drain, less accuracy required, and multiple lives. As a bonus, it make the map impossible to read!

Reviewer: ```suggestion
  - Larger circles, more forgiving HP drain, less accuracy required, and multiple lives. As a bonus, it makes the map impossible to read!
```

  - Score Multiplier: 0.85x

> - ![No Fail Icon](mod-icon/no-fail-mod.png) No Fail
>   - You can't fail, no matter what.
>     - Make it impossoble to fail.
>   - Score Multiplier: 0.50x

Reviewer: ```suggestion
- ![No Fail Icon](mod-icon/no-fail-mod.png) No Fail
  - You can't fail, no matter what.
    - Makes it impossible to fail.
  - Score Multiplier: 0.50x
```

- ![Half Time Icon](mod-icon/half-time-mod.png) Half Time
  - Less Zoom...
    - Decreases speed .75x.
  - Score Multiplier: 0.30x
  - Click the icon again for `Daycore Mod` : Whoaaaaa...
    - Decreases speed .75x and lowers audio pitch.

### difficulty increase

- ![Hard Rock Icon](mod-icon/hard-rock-mod.png) Hard Rock
  - Everything just got a bit harder...
  - Score Multiplier: 1.06x

- ![Sudden Death Icon](mod-icon/sudden-death-mod.png) Sudden Death
  - Miss and fail.
    - If you miss, you fail.
  - Click the icon again for `Perfect Mod` : SS or quit.

- ![Double Time Icon](mod-icon/double-time-mod.png) Double Time
  - Zoooooooooom...
    - Increased speed 1.50x.
  - Score Multiplier: 1.12x
  - Click the icon again for `Nightcore Mod` : Uguuuuuuuu...
    - Increases speed 1.50x and raises audio pitch.

### automation

- ![Autoplay Icon](mod-icon/autoplay-mod.png) Autoplay
  - Watch a perfect automated play through the song.

### conversion

- ![Random Icon](mod-icon/random-mod.png) Random
  - Shuffle around the notes!

> - ![Difficulty Adjust Icon](mod-icon/difficulty-adjust-mod.png) Difficulty Adjust
>   - Override a beatmap's difficulty settings.
>   - You can override *HP Drain*, *Accuracy*, *Paddle Size*, *Approach Rate* and *Extended Limits*

Reviewer: ```suggestion
- ![Difficulty Adjust Icon](mod-icon/difficulty-adjust-mod.png) Difficulty Adjust
  - Override a beatmap's difficulty settings.
  - You can override *Circle Size*, *HP Drain*, *Accuracy*, *Approach Rate* and *Extended Limits*
```

### fun

- ![Wind Up Icon](mod-icon/wind-up-mod.png) Wind Up
  - Can you keep up?
    - Slowly increases speed from 1.00x to 1.50x throughout the map and raises the pitch alongside the speed.
  - The song speed will be getting faster and faster
  - Click the icon again will be `Wind Down Mod` : Slooow doooown...
    - Slowly decreases speed from 1.00x to 0.75x throughout the map and lowers the pitch alongside the speed.

# settings

Player can adjust

> - `Show Input Overlay` by toggle button
> - `Show Kiai Visualizer` by toggle button
> - `Hit Circle distance from screen center` by slider
> - `Hit Circle gap` by slider
> - `Judgement distance from screen center` by slider
> - `Judgement gap` by slider
> - `Input Overlay distance from screen center` by slider
> - `Input Overlay gap` by slider
> - `Kiai Visualizer distance from screen center` by slider
> 

Reviewer: ```suggestion
- `Show Input Overlay` by a toggle button
- `Show Kiai Visualizer` by a toggle button
- `Hit Circle distance from screen center` by a slider
- `Hit Circle gap` by a slider
- `Judgement distance from screen center` by a slider
- `Judgement gap` by a slider
- `Input Overlay distance from screen center` by a slider
- `Input Overlay gap` by a slider
- `Kiai Visualizer distance from screen center` by a slider

```

> # demo
> 
> {{< youtube id="hWjG0W7EiAE" >}}

Reviewer: ```suggestion
# demo

{{< youtube id="hWjG0W7EiAE" >}}
```
```suggestion
# preview

{{< youtube id="hWjG0W7EiAE" >}}
```
